draft-regext-rdap-extensions.md

```diff
@@ -331,6 +331,35 @@ following:
  - whether the extension itself should define how versioning is
    handled within the extension documentation.
 
+# Multiple Extension Identifiers
+
+Extension specifications have customarily defined only one extension identifier. However,
+there is no explicit limit on the number of extension identifiers that may be defined in
+a single extension specification. One example is [@?I-D.ietf-regext-rdap-rir-search].
+
+[@!RFC9083, Section 4.1] offers the following guidance on using the extension identifiers
+in RDAP responses:
+
+    A response to a "help" request will include identifiers for all of the specifications
+    supported by the server. A response to any other request will include only identifiers 
+    for the specifications used in the construction of the response.
+
+A strict interpretation of this wording where "construction of the response" only refers
+to the JSON syntax would rule out the usage of (#profiles_and_markers) extension IDs which
+are in common use in RDAP. This document updates the guidance. For responses to queries
+other than "help", a response MUST include in the `rdapConformance` array only those extension
+identifiers necessary for a client to deserialize the JSON and understand the semantic meaning
+of the content within the JSON, and all extensions identifiers MUST be free from conflict in
+both their syntactic and semantic meaning. If an extensions specification defines multiple
+extension identifiers, the extension definition MAY specify that one or more of the identifiers
+may be returned in non-"help" responses so long as the identifiers do not conflict.
```

Comment:
I think this text is fine, but one point here is that the last sentence appears to conflict with the earlier "MUST include" requirement.  With RIR search (e.g.), could we have said "servers should return all extension identifiers in all responses that make use of any part of the specification"?  That would appear to be fine as far as the last sentence is concerned (since the identifiers don't conflict), but not fine with respect to the earlier "MUST include" requirement.

Reply:
Should we just strike that last sentence?

Reply:
I think that works, yep.  (If a specification needs more flexibility as to how its identifiers work, I think that will fit within "understand the semantic meaning of the content".)

Reply:
Agreed.

Reply:
I'll make the title change in the main branch when I am doing the date and draft name changes.